src/test/fixtures/fake-suggest-change-store.ts

```diff
@@ -0,0 +1,75 @@
+import { ISuggestChangeStore } from '../../lib/types/stores/suggest-change-store';
+import { ISuggestChange, ISuggestChangeset } from '../../lib/types/model';
+import { PartialSome } from '../../lib/types/partial';
+import User from '../../lib/types/user';
+
+export default class FakeSuggestChangeStore implements ISuggestChangeStore {
+    suggestChanges: ISuggestChangeset[] = [];
+
+    async get(id: number): Promise<ISuggestChangeset> {
+        const change = this.suggestChanges.find((c) => c.id === id);
+        return Promise.resolve(change);
+    }
+
+    async count(): Promise<number> {
+        return Promise.resolve(0);
+    }
+
+    // eslint-disable-next-line no-unused-vars,@typescript-eslint/no-unused-vars
+    async delete(id: number): Promise<void> {
+        return Promise.resolve(undefined);
+    }
+
+    addChangeToSet(
+        change: ISuggestChange,
+        changeSetID: number,
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
+        user: Partial<Pick<User, 'username' | 'email'>>,
+    ): Promise<void> {
+        const changeSet = this.suggestChanges.find((s) => (s.id = changeSetID));
+        changeSet.changes.push(change);
+        return Promise.resolve();
+    }
+
+    getForEnvironment(environment: string): Promise<ISuggestChangeset[]> {
+        return Promise.resolve(
+            this.suggestChanges.filter(
+                (changeSet) => (changeSet.environment = environment),
```

Comment:
Am I missing something or this should be `===` and also for other methods  ?

Reply:
Good catch, thanks